client/sites-dashboard/components/test/sites-site-item-thumbnail.tsx

```diff
@@ -85,4 +87,9 @@ function defineCommonSiteInitialTests() {
 		render( <SiteItemThumbnail site={ testSite } /> );
 		expect( screen.getByLabelText( 'Site Icon' ) ).toBeEmptyDOMElement();
 	} );
+
+	test( 'shows "Coming soon" tile if site has not launched', () => {
+		render( <SiteItemThumbnail site={ makeTestSite( { title: '', is_coming_soon: true } ) } /> );
+		expect( screen.getByTestId( 'site-coming-soon' ) ).toBeInTheDocument();
```

Comment:
The spirit behind `testing-library` is to only resort to test IDs when necessary but to prefer more user-centric assertions. Will the `screen.getByText( 'Coming Soon' )` work in this case?

Reply:
I tried this first but `testing-library` can't find text inside of `tspan` elements by the looks of things. However, for the additional tests that I added, I also included `<title>` in the `SVG` which `testing-library` can find and encourages you to add via its API! So I will make this test use the `getByTitle` and remove the ID. Good spot!